src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -22,18 +22,34 @@
     </span>
   </div>
   <div class="card-body fill-plain">
-    <div>
+    <form #courseForm="ngForm" (ngSubmit)="onSaveCourse()">
       <div class="form-row form-group">
         <label class="col-sm-3">Course ID:</label>
         <div class="col-sm-3">
-          <input class="col-xs-12 form-control" maxlength="40" tabindex="1" [(ngModel)]="course.courseId" disabled>
+          <input class="col-xs-12 form-control" type="text" tabindex="1"
+                 #courseId="ngModel" [(ngModel)]="course.courseId" name="courseId"
+                 [ngClass]="{'invalid' : courseId.invalid && (courseId.touched || courseId.dirty),
```

Comment:
Should we standardise this in other forms? For example, right now the instructor course edit student page does not have this indication bar.

Reply:
Makes sense, I'll follow the implementation in #9633